### ISSUE TITLE
Dumb AP?

### ISSUE THREAD

**Issue by eduperez** [ex-ml/Hassio-Access-Point#8]
Would it be possible to add an option to just bridge WLAN0 to LAN0, without any DHCP or routing, so the device becomes a "dumb access point", please?

**Comment by mattlongman** [comment]
Hi @eduperez - I'm looking into this.

**Comment by G2G2G2G** [comment]
Yea this would be amazing
@mattlongman should be able to disable dhcpd, create a bridge device and add it to both interfaces? eth0 and wlan0 I am guessing for 99% of people
https://www.raspberrypi.com/documentation/computers/configuration.html#bridging

technically be possible with 2 wifi dongles as well if you want the pi to be an extender I guess (would save that for later)

**Comment by stegno** [comment]
I also looking for way to do a bridged dumb AP, since its done, AP could be a part of wifi coverage withing home (if same SSID and security) with normal roaming between other APs. Now it has to change IPs and data transmitting stops for a much more time..

**Comment by HLFCode** [comment]
A wifi/ethernet bridge would solve my problems too. Can you share progress? Happy to collaborate.
I can get a pi to bridge but still on steep learning curve with Docker...
Thanks @mattlongman for a great addon

**Comment by G2G2G2G** [comment]
@HLFCode agreed, the docker nightmare that HA imposes made me just rewrite my own home automation with uglier UI

**Comment by xonestonex** [comment]
My PR https://github.com/home-assistant/operating-system/pull/2304 got merged. 
With HAOS 10 it will be possible to create a hotspot and bridge it with the ethernet interface directly within NetworkManager. Some nmcli commands and one iptables rule in the addin will be enough to create such dumb AP.

**Comment by HLFCode** [comment]
Fantastic @xonestonex .
Do I have to upgrade from HAOS 9.4 to 10 before I can use it?

**Comment by voed** [comment]
Can someone explain how to configure it as "Dumb AP"? @xonestonex @eduperez maybe you have an example how to make the bridge?

**Comment by HLFCode** [comment]
Yes please

**Comment by stav242** [comment]
+1 - im looking to bridge wlan0 and end0, i cant find details of how to do this on HASSOS 13.2.

any suggestions ?

**Comment by ex-ml** [comment]
Is there still a need for this?
